app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java

```diff
@@ -105,6 +109,14 @@ public void onDestroyView() {
         super.onDestroyView();
     }
 
+    @Override
+    public void setUserVisibleHint(boolean isVisibleToUser) {
```

Comment:
Redundant code: Same as with mIsVisibleToUser. This should maybe be put into `BaseFrament`.